doc/releases/changelog-dev.md

```diff
@@ -1,1120 +0,0 @@
-:orphan:
```

Comment:
This file had to be kept so that the Shpinx check passes. It doesn't contain anything.

Reply:
@antalszava I think it can now be deleted given a92b61a. Although, I wonder the best approach in order to minimize merge conflicts when later merging this branch back into master

Reply:
Thanks. Perhaps leaving it in should also be okay, as the `changelog-dev.md` in master also has it?

@josh146 let me know if doing `git mv doc/releases/changelog-dev.md doc/releases/changelog-0.19.0.md` would be preferable such that GitHub highlights the diff better. Right now there's a completely new `doc/releases/changelog-0.19.0.md` file with ~1200 lines of changes.